src/request.rs

```diff
@@ -125,7 +125,16 @@ impl SecretType for SecretKey {
 
     async fn policies(&self, db: &KbsDb) -> Vec<policy::Policy> {
         match db.get_secret_policy(&self.request.id).await {
-            Ok(policy) => vec![policy],
+            Ok(popt) => match popt {
+                Some(policy) => vec![policy],
+                None => {
+                    eprintln!(
```

Comment:
You should use `info!` for this instead (or possibly `warning!`

Reply:
... and also below for other `eprintln!()` calls.